app/views/crops/index.html.haml

```diff
@@ -1,5 +1,6 @@
 - content_for :title, "Crops"
-
+  - if can? :wrangle, Crop
+  = link_to 'Wrangle Crops', wrangle_crops_path, :class => 'btn btn-primary'
```

Comment:
I think you need to indent this line one step further, so it's within the if-block.